Fixed typos and grammar issues (#160)
aws/eks-anywhere: docs/content/en/docs/reference/vsphere/vsphere-ovas.md

## Using vCenter Web User Interface

1. Right-click on your Datacenter, and select *Deploy OVF Template*
   ![Import ova drop down](/images/ss1.jpg)
1. Select an OVF template using URL or selecting a local OVF file, and click on *Next*. If you are unable to select an
   OVF template using URL, download the file and use Local file option.
   
   Note: If you are using Bottlerocket OVAs, please select local file option.
   ![Import ova wizard](/images/ss2.jpg)
1. Select a folder where you want to deploy your OVF package (most of our OVF templates are under SDDC-Datacenter
   directory), and click on *Next*. You cannot have an OVF template with the same name in one directory. For workload
   VM templates, leave the Kubernetes version in the template name for reference. A workload VM template will

>    support at least one prior Kubernetes minor versions.

Reviewer: ```suggestion
   support at least one prior Kubernetes minor version.
```

   ![Import ova wizard](/images/ss3.jpg)
1. Select any compute resource to run (from cluster-1, 10.2.34.5, etc.) the deployed VM, and click on *Next*
   ![Import ova wizard](/images/ss4.jpg)
1. Review the details and click *Next*.
1. Accept the agreement, and click *Next*.

> 1. Select the appropriate storage (e.g., “WorkloadDatastore“), and click *Next*.

Reviewer: ```suggestion
1. Select the appropriate storage (for example, “WorkloadDatastore“), and click *Next*.
```

> 1. Select destination network (e.g., “sddc-cgw-network-1”), and click *Next*.

Reviewer: ```suggestion
1. Select destination network (for example, “sddc-cgw-network-1”), and click *Next*.
```

1. Finish. 
1. Snapshot the VM. Right-click on the imported VM, and select Snapshots -> Take Snapshot... 
   (It is highly recommended that you snapshot the VM. This will reduce the time it takes to provision
   machines and cluster creation will be faster. If you prefer not to take a snapshot, skip to step 13)
![Import ova wizard](/images/ss6.jpg)

> 1. Name your template (e.g., "root") and click *Create*.

Reviewer: ```suggestion
1. Name your template (for example, "root") and click *Create*.
```

    govc vm.markastemplate <name of new VM>
    ```

## Important Additional Steps to Tag the OVA

### Using vCenter UI

#### Tag to indicate OS family

1. Select the template that was newly created in the steps above, and navigate to *Summary* -> *Tags*.
   ![Import ova wizard](/images/ss10.jpg)
1. Click *Assign* -> *Add Tag* to create a new tag and attach it.
   ![Import ova wizard](/images/ss11.jpg)
1. Name the tag *os:ubuntu* or *os:bottlerocket*.
   ![Import ova wizard](/images/ss12.jpg)

#### Tag to indicate eksd release
1. Select the template that was newly created in the steps above and navigate to *Summary* -> *Tags*.
   ![Import ova wizard](/images/ss10.jpg)
1. Click *Assign* -> *Add Tag* to create a new tag and attach it.
   ![Import ova wizard](/images/ss11.jpg)

> 1. Name the tag *eksdRelease:{EKS-D release for the selected ova}* (for example *eksdRelease:kubernetes-1-20-eks-5* for the 1.20 OVA). You can find the rest of the EKS-D releases in the previous [section](#import-an-ovaovf-template-to-vsphere). If it's the first time you add an `eksdRelease` tag, you would need to create the category first. Click on "Create New Category", and name it `eksdRelease`.

Reviewer 2: ```suggestion
1. Name the tag *eksdRelease:{EKS-D release for the selected ova}* (for example, *eksdRelease:kubernetes-1-20-eks-5* for the 1.20 OVA). You can find the rest of the EKS-D releases in the previous [section](#import-an-ovaovf-template-to-vsphere). If it's the first time you add an `eksdRelease` tag, you would need to create the category first. Click on "Create New Category", and name it `eksdRelease`.
```

Author: Fix in next revision

> 1. Name the tag *eksdRelease:{EKS-D release for the selected ova}* (for example *eksdRelease:kubernetes-1-20-eks-5* for the 1.20 OVA). You can find the rest of the EKS-D releases in the previous [section](#import-an-ovaovf-template-to-vsphere). If it's the first time you add an `eksdRelease` tag, you would need to create the category first. Click on "Create New Category", and name it `eksdRelease`.

Reviewer: ```suggestion
1. Name the tag *eksdRelease:{EKS-D release for the selected ova}* (for example *eksdRelease:kubernetes-1-20-eks-5* for the 1.20 OVA). You can find the rest of the EKS Distro releases in the previous [section](#import-an-ovaovf-template-to-vsphere). If it's the first time you add an `eksdRelease` tag, you would need to create the category first. Click on "Create New Category", and name it `eksdRelease`.
```

   ![Import ova wizard](/images/ss13.png)

### Using govc

#### Tag to indicate OS family

1. Create tag category.

```
govc tags.category.create -t VirtualMachine os
```

> 1. Create tags *os:ubuntu* and *os:bottlerocket*.

Reviewer: ```suggestion
1. Create tags `os:ubuntu` and `os:bottlerocket`.
```
I think italics (*) should mostly be used for things that are substitutable, not literal.

```
govc tags.create -c os os:bottlerocket
govc tags.create -c os os:ubuntu
```

> 1. Attach newly-created tags to the template.

Reviewer: ```suggestion
1. Attach newly created tags to the template.
```

```
govc tags.attach os:bottlerocket <Template Path>
govc tags.attach os:ubuntu <Template Path>
```
1. Verify tags are attached to the template.

```
govc tags.ls <Template Path> 
```

#### Tag to indicate EKS-D release
1. Create tag category.
```
govc tags.category.create -t VirtualMachine eksdRelease
```
2. Create the proper EKS-D release tag, depending on your template. You can find the EKS-D releases in the previous [section](#import-an-ovaovf-template-to-vsphere). For example, *eksdRelease:kubernetes-1-20-eks-5* for the 1.20 template.
```
govc tags.create -c eksdRelease eksdRelease:kubernetes-1-20-eks-2
```
3. Attach newly-created tag to the template.
```
govc tags.attach eksdRelease:kubernetes-1-20-eks-2 <Template Path>
```
4. Verify tag is attached to the template.

```
govc tags.ls <Template Path> 
```
{{% alert title="Note" color="primary" %}}

> If the tags above are not applied as shown exactly, EKS-A template validations will fail and CLI will abort.

Reviewer: ```suggestion
If the tags above are not applied as shown exactly, EKS Anywhere template validations will fail and CLI will abort.
```
